pyproject.toml

```diff
@@ -103,6 +106,7 @@ docs = [
     "myst-parser == 0.18.0",
     "nlp_primitives >= 2.3.0",
     "autonormalize >= 2.0.1",
+    "featuretools_sql >= 0.0.1",
```

Comment:
curious -- why can't we use `featuretools[sql]` here?

Reply:
Hmm, I guess we could